go/client/simplefs_windows.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package client
+
+import (
+	"context"
+	"errors"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func doSimpleFSRemoteGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, path keybase1.Path) ([]keybase1.Path, error) {
+
+	var returnPaths []keybase1.Path
+	directory := filepath.ToSlash(filepath.Dir(path.Kbfs()))
+	base := filepath.Base(path.Kbfs())
+
+	// We know the filename has wildcards at this point.
+	// kbfs list only works on directories, so build a glob from a list result.
+
+	g.Log.Debug("doSimpleFSRemoteGlob %s", path.Kbfs())
+
+	if strings.ContainsAny(filepath.Base(directory), "?*[]") == true {
```

Comment:
I guess here's the case I'm thinking about. Say I have a file at this literal path (with real stars in it): `*/*.txt`. In the current implementation, it's impossible to address that path at all. We could just interpret everything before the basename literally, in which case the right way to address that path would be `*/\*.txt`. That's unfortunately inconsistent though; you have to escape the second star, but you must not escape the second one.

I think the most correct way for us to tell people to represent that path is `\*/\*.txt`. That way the rule is consistent: "escape all your literal stars." We then still have the option of saying unescaped stars in the dirname cause an error. (And if we want to add support for stars everywhere in the future, it will be backwards-compatible.) Is that doable?